src/components/personkort/PersonkortHeader/UtdatertUtbetalingsinfoPopoverButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useRef, useState } from "react";
+import { Button, Popover } from "@navikt/ds-react";
+import { ExclamationmarkIcon } from "@navikt/aksel-icons";
+
+export default function UtdatertUtbetalingsinfoPopoverButton() {
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      <Button
+        icon={<ExclamationmarkIcon title="Rediger" />}
```

Comment:
Skal det stå "Rediger" her?

Reply:
Nope! Gikk litt for fort unna denne her😅 Bra catch! 🎣